src/core/Evaluate.re

```diff
@@ -22,23 +22,28 @@ type warning = {
 
 [@deriving show]
 type blockContent =
+  | BlockStart
   | BlockSuccess{
       msg: string,
       warnings: list(warning),
+      stdout: string,
```

Comment:
Any particular reasons why you move them here? It's fine by me, just curious.

Edit: nvm. you need it for BlockStart

Reply:
Ah ya exactly - I was either going to just set it to `""`, but I noticed `warnings` were shared between `BlockSuccess` and `BlockError`, so I thought it'd be weird to have `warnings` common but not `stdout`. Let me know if you need if you'd prefer a different design though, happy to change it